2022-01-28 11:07:55,065 INFO    MainThread:19423 [wandb_setup.py:_flush():69] setting env: {}
2022-01-28 11:07:55,065 INFO    MainThread:19423 [wandb_setup.py:_flush():69] setting login settings: {}
2022-01-28 11:07:55,066 INFO    MainThread:19423 [wandb_init.py:_log_setup():334] Logging user logs to /Users/gdegournay/PycharmProjects/Udacity/MLOps/MLOps_ex/reproducible_workflow/exercices/lesson-2-data-exploration-and-preparation/exercises/exercise_4/starter/wandb/run-20220128_110755-2pm8lugb/logs/debug.log
2022-01-28 11:07:55,066 INFO    MainThread:19423 [wandb_init.py:_log_setup():335] Logging internal logs to /Users/gdegournay/PycharmProjects/Udacity/MLOps/MLOps_ex/reproducible_workflow/exercices/lesson-2-data-exploration-and-preparation/exercises/exercise_4/starter/wandb/run-20220128_110755-2pm8lugb/logs/debug-internal.log
2022-01-28 11:07:55,067 INFO    MainThread:19423 [wandb_init.py:_jupyter_setup():285] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x1a4446370>
2022-01-28 11:07:55,067 INFO    MainThread:19423 [wandb_init.py:init():367] calling init triggers
2022-01-28 11:07:55,068 INFO    MainThread:19423 [wandb_init.py:init():372] wandb.init called with sweep_config: {}
config: {}
2022-01-28 11:07:55,068 INFO    MainThread:19423 [wandb_init.py:init():416] starting backend
2022-01-28 11:07:55,068 INFO    MainThread:19423 [backend.py:_multiprocessing_setup():69] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2022-01-28 11:07:55,078 INFO    MainThread:19423 [backend.py:ensure_launched():123] starting backend process...
2022-01-28 11:07:55,092 INFO    MainThread:19423 [backend.py:ensure_launched():127] started backend process with pid: 19433
2022-01-28 11:07:55,094 INFO    MainThread:19423 [wandb_init.py:init():421] backend started and connected
2022-01-28 11:07:55,111 INFO    MainThread:19423 [wandb_init.py:init():461] updated telemetry
2022-01-28 11:07:55,111 INFO    MainThread:19423 [wandb_init.py:init():480] communicating current version
2022-01-28 11:07:55,566 INFO    MainThread:19423 [wandb_init.py:init():485] got version response 
2022-01-28 11:07:55,566 INFO    MainThread:19423 [wandb_init.py:init():493] communicating run to backend with 30 second timeout
2022-01-28 11:08:25,571 INFO    MainThread:19423 [interface.py:_communicate_run():635] couldn't get run from backend
2022-01-28 11:08:25,572 ERROR   MainThread:19423 [wandb_init.py:init():497] backend process timed out
2022-01-28 11:08:25,572 ERROR   MainThread:19423 [wandb_init.py:init():506] encountered error: Error communicating with wandb process
try: wandb.init(settings=wandb.Settings(start_method='fork'))
or:  wandb.init(settings=wandb.Settings(start_method='thread'))
For more info see: https://docs.wandb.ai/library/init#init-start-error
2022-01-28 11:09:13,270 INFO    MainThread:19423 [wandb_init.py:teardown():185] tearing down wandb.init
